### PR TITLE
Use utf8 encoding everywhere

### PR DESCRIPTION
**Pull request by ChristophWurst**
PSR2 likes that and so do I

### CODE REVIEW

File: apps/settings/templates/settings/empty.php [hunk 1]
```diff
@@ -21,4 +21,4 @@
  *
  */
 
-	# used for Personal/Additional settings as fallback for legacy settings
```
Comment (ChristophWurst) [inline apps/settings/templates/settings/empty.php:24]: can't even tell what changed here. might need to hexedit it :shrug:

**Review by skjnldsv — APPROVED** [review]

File: apps/settings/templates/settings/empty.php (hunk 1 above)
Comment (skjnldsv) [reply apps/settings/templates/settings/empty.php:24]: Nah, changed the EOF

**Review by rullzer — APPROVED** [review]

**Comment by gary-kim** [comment]
@skjnldsv I'm assuming you were trying to add `4. to release` rather than `14-feedback`? Otherwise, one of your customers are on a ridiculously old version.

**Comment by skjnldsv** [comment]
> @skjnldsv I'm assuming you were trying to add `4. to release` rather than `14-feedback`? Otherwise, at least one of your customers really hates upgrading.

oups :see_no_evil: 
Thanks for spotting this!

**Pull request merged**